src/WebApiTestApplication/Scripts/Endpoints/Service.ts

```diff
@@ -17,12 +20,29 @@ namespace Endpoints {
             return call.then(response => response.data);
         }
     
+        static callCached<TView>(endpoint: IEndpoint, data) {
+            if (this.endpointCache[endpoint.toString()] == null) {
```

Comment:
I'd prefer to extract `endpoint.toString()` to a variable here. Perhaps `cacheKey`? And reuse below?